tikv-client/src/main/java/com/pingcap/tikv/TiBatchWriteUtils.java

```diff
@@ -15,15 +15,31 @@
 
 package com.pingcap.tikv;
 
+import com.pingcap.tikv.key.IndexKey;
 import com.pingcap.tikv.key.Key;
 import com.pingcap.tikv.key.RowKey;
+import com.pingcap.tikv.meta.TiIndexInfo;
 import com.pingcap.tikv.meta.TiTableInfo;
 import com.pingcap.tikv.region.TiRegion;
 import java.util.ArrayList;
 import java.util.List;
 
 public class TiBatchWriteUtils {
 
+  public static List<TiRegion> getRegionByIndex(
+      TiSession session, TiTableInfo table, TiIndexInfo index) {
+    ArrayList<TiRegion> regionList = new ArrayList<>();
+    Key min = IndexKey.toIndexKey(table.getId(), index.getId());
+    Key max = min.nextPrefix();
+
+    while (min.compareTo(max) < 0) {
+      TiRegion region = session.getRegionManager().getRegionByKey(min.toByteString());
+      regionList.add(region);
+      min = Key.toRawKey(region.getEndKey());
```

Comment:
somewhat dangerous if region splits or changes...

Reply:
This only used for testing, I believe.

Reply:
`getRegionByIndex` and `getRegionByTable` is almost the same. Should we move them to `core/src/test` ?

Reply:
as u said, `getRegionByIndex` is only used for testing, if we put here, other developer may use it as a mistake

Reply:
My point is that `getRegionByIndex` is identical `getRegionByTable`, what should I do for `getRegionByTable`. `getRegionByTable` is used in `TiBatchWrite`.